stdlib/3/typing.pyi

```diff
@@ -535,6 +535,7 @@ def cast(tp: Type[_T], obj: Any) -> _T: ...
 
 # NamedTuple is special-cased in the type checker
 class NamedTuple(tuple):
+    _field_types = ...  # type: collections.OrderedDict[str, Tuple[str, Type]]
```

Comment:
It's actually an `OrderedDict[str, Type[Any]]`.

Reply:
You're right, sorry. Fixed now.